### ISSUE TITLE
Option to show answers for different categories. [Feature Request]

### ISSUE THREAD

**Issue by DrZoddiak** [SimonFlash/TriviaQuest#3]
Currently there is only 1 configuration option for showing answers, but I would like the ability to not show answers for Questions, but still show the answers for scrambles, and completions.

**Comment by SimonFlash** [comment]
Good point. This would likely be migrated into a configuration setting at the pack level (if not for each question directly).
